nebula_common/include/nebula_common/hesai/hesai_common.hpp

```diff
@@ -34,6 +34,7 @@ namespace drivers
 /// @brief struct for Hesai sensor configuration
 struct HesaiSensorConfiguration : public LidarConfigurationBase
 {
+  std::string multicast_ip;
```

Comment:
```suggestion
  std::string multicast_ip{};
```

Reply:
Clang-Tidy flags this as a [warning](https://clang.llvm.org/extra/clang-tidy/checks/readability/redundant-member-init.html) because non-primitives always call their default constructor on init anyway, even without `{}`.